ncm-spma/pom.xml

```diff
@@ -195,12 +224,32 @@
                 </sources>
                 <directoryIncluded>false</directoryIncluded>
               </mapping>
+              <mapping>
+                <directory>/usr/sbin</directory>
+                <filemode>755</filemode>
+                <username>root</username>
+                <groupname>root</groupname>
+                <directoryIncluded>false</directoryIncluded>
+                <sources>
+                  <source>
+                    <location>${project.build.directory}/sbin</location>
+                    <includes>
+                      <include>spma</include>
+                    </includes>
+                  </source>
+                </sources>
+                <configuration>false</configuration>
+                <documentation>false</documentation>
+              </mapping>
             </mappings>
             <requires>
               <require>yum &gt;= 3.2.29</require>
               <require>yum-utils &gt;= 1.1.30</require>
               <require>yum-versionlock</require>
               <require>yum-priorities</require>
+              <require>yum-fastestmirror</require>
+              <require>yum-list-data</require>
+              <require>redhat-release</require>
```

Comment:
We should likely be good with system-release. Assuming centos-release provides it as well?